veloroutes_voies_vertes/test/test_export.py

```diff
@@ -47,6 +47,15 @@ def test_export_1file(self):
         path = plugin_test_data_path('N_3V_PORTION_L_066.shp')
         self.assertTrue(os.path.isfile(path))
 
+        for i in range(15):
```

Comment:
Can you update the test just above as well ? It's a duplicate with `1`, already included in this range.

Can you add a test for the `export_all` as well ? Better to have a test for all Processing lgorithm when we can.

Reply:
Now I have an error in the `export_all` test